### PR TITLE
Swaggerify /account

### PR DESCRIPTION
**Pull request by illicitonion**

### CODE REVIEW

File: api/client-server/administrative_contact.yaml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+securityDefinitions:
+  accessToken:
+    type: apiKey
+    description: The user_id or application service access_token
+    name: access_token
+    in: query
+paths:
+  "/account/password":
+    post:
+      summary: Changes a user's passwordl
```
Comment (Kegsay) [inline api/client-server/administrative_contact.yaml]: `passwordl`

File: api/client-server/administrative_contact.yaml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+            properties:
+              threepids:
+                type: array
+                items:
+                  type: object
+                  title: Third party identifier
+                  properties:
+                    medium:
+                      type: string
+                      description: The medium of the third party identifier.
```
Comment (Kegsay) [inline api/client-server/administrative_contact.yaml]: I feel this should be marked as an enum of just `["email"]` currently.

File: api/client-server/administrative_contact.yaml [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+      summary: Adds contact information to the user's account.
+      description: Adds contact information to the user's account.
+      security:
+        - accessToken: []
+      parameters:
+        - in: body
+          name: body
+          schema:
+            type: object
+            description: "Stuff"
```
Comment (Kegsay) [inline api/client-server/administrative_contact.yaml]: Nice description :) - and `:114`

File: api/client-server/administrative_contact.yaml (hunk 2 above)
Comment (illicitonion) [inline api/client-server/administrative_contact.yaml]: It's not our job to tell people they shouldn't use SMS for password reset

File: api/client-server/administrative_contact.yaml [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+                  sid:
+                    type: string
+                    description: The session identifier given by the Identity Server.
+                required: ["client_secret", "id_server", "sid"]
+              bind:
+                type: boolean
+                description: |-
+                  Whether the home server should also bind this third party
+                  identifier to the account's Matrix ID with the passed identity
+                  server. Default: false.
```
Comment (Kegsay) [inline api/client-server/administrative_contact.yaml]: Backticks on `false` please.

File: api/client-server/administrative_contact.yaml (hunk 1 above)
Comment (illicitonion) [inline api/client-server/administrative_contact.yaml]: Done

File: api/client-server/administrative_contact.yaml (hunk 2 above)
Comment (Kegsay) [inline api/client-server/administrative_contact.yaml]: I would like to be in a place where we can say:
 - Medium: `email`. Address: "the email address"
 - Medium: `sms`. Address: "the MSISDN"

We are not in that place. We only support `email`. If and when we add more, we can add it then.

File: api/client-server/administrative_contact.yaml (hunk 3 above)
Comment (illicitonion) [inline api/client-server/administrative_contact.yaml]: Done

(Oops)

File: api/client-server/administrative_contact.yaml (hunk 4 above)
Comment (illicitonion) [inline api/client-server/administrative_contact.yaml]: Done

File: api/client-server/administrative_contact.yaml (hunk 2 above)
Comment (illicitonion) [inline api/client-server/administrative_contact.yaml]: Done

**Comment by Kegsay** [comment]
LGTM

**Pull request merged**